courses/models.py

```diff
@@ -697,6 +706,34 @@ class Meta:
     def get_certified_object_id(self):
         return self.program_id
 
+    def get_courseware_object_id(self):
+        """Gets the program id"""
+        return self.program_id
+
+    @property
+    def link(self):
+        """
+        Get the link at which this certificate will be served
+        Format: /certificates/program/<uuid>/
+        Example: /certificates/program/93ebd74e-5f88-4b47-bb09-30a6d575328f/
+        """
+        return "/certificates/program/{}/".format(str(self.uuid))
+
+    @property
+    def start_end_dates(self):
```

Comment:
It would be good if we add tests for the dates queries for both `ProgramCertificate` and `CourserunCertificates`.

Reply:
Sure